### ISSUE TITLE
[Bug] Migration from v0.4.1 to 0.6.6 return a mMap Object - Android

### ISSUE THREAD

**Issue by alessioemireni** [ammarahm-ed/react-native-mmkv-storage#181]
**Describe the bug**
After migration from v0.4.1 to v0.6.6 the method `getArrayAsync`  returns a `mMap` Object containing the Object of the previous saved data.

This problem occurs only with Android devices.

Also when I save other data with  `setArrayAsync` the database is broken!

**To Reproduce**
1. Create an example app with `react-native-mmkv-storage` v0.4.1;
2. Save some data using `setArrayAsync` like this (Array of Object):
    `await MMKVStorage.setArrayAsync('array', new Array([{'test'; 'test'}, {'test': 'test'}]))`
3. Migrate to `react-native-mmkv-storage` v0.6.6;
4. Get the data like this:
   ` let array = await MMKVStorage.getArrayAsync('array');`
5. The Array contains a `mMap` Object of Objects of the previous migrated data;

**Expected behavior**
The result should be like this:

![Screenshot 2021-11-04 at 16 53 40](https://user-images.githubusercontent.com/9269526/140387008-38b082b9-c1b0-4b81-8a28-0b4ec23a65b3.png)

**Screenshots**
The wrong result:

![Screenshot 2021-11-04 at 18 10 58](https://user-images.githubusercontent.com/9269526/140386719-46bd1a61-3374-4e94-8dc3-b5ea02977ed0.png)

**Platform Information:**
 - OS: [Android]
 - React Native Version [0.65.1]
 - Library Version [0.6.6]

**Comment by ammarahm-ed** [comment]
Seems to be an issue with the migrator. I will try to debug this.

**Comment by alessioemireni** [comment]
Hi @ammarahm-ed . Any news on this issue?

**Comment by ammarahm-ed** [comment]
@alessioemireni I have fixed the issue. You can test with the master branch installed.

```
yarn add https://github.com/ammarahm-ed/react-native-mmkv-storage.git
```

**Comment by alessioemireni** [comment]
Hi @ammarahm-ed I've tested the migration and now it works correctly.

Could you please create a release?

Many thx

**Comment by alessioemireni** [comment]
Hi @ammarahm-ed . Any news about a release date? thx

**Comment by alessioemireni** [comment]
@ammarahm-ed news about this issue? The update is critical due to the fact that the old version of MMKV uses JCenter as repository.

**Comment by ammarahm-ed** [comment]
@alessioemireni i will release soon. need to test a couple of things.

**Comment by alessioemireni** [comment]
Hi @ammarahm-ed any updates on new release?

**Comment by ammarahm-ed** [comment]
Released v0.6.7
